server/config/toml.go

```diff
@@ -100,4 +100,8 @@ certificate-path = "{{ .TLS.CertificatePath }}"
 
 # Key path defines the key.pem file path for the TLS configuration.
 key-path = "{{ .TLS.KeyPath }}"
-` + memiavlcfg.DefaultConfigTemplate
+`
+
+// DefaultEvmosConfigTemplate defines the configuration template for the full Evmos chain configuration
+// which contains the EVM RPC configuration plus the MemIAVL configuration.
+const DefaultEvmosConfigTemplate = EVMConfigTemplate + memiavlcfg.DefaultConfigTemplate
```

Comment:
I would move this one in the `config.go` to keep here just the possible configuration and use the config file to combine them

Reply:
good idea, that would be cleaner 👍